lang/de.json

```diff
@@ -741,6 +741,27 @@
         "ShareaholicTagSiteIdTitle": "Shareaholic Site-ID",
         "ShowAdvancedSettings": "Erweiterte Einstellungen anzeigen",
+
+        "SiteWithoutDataReactIntro": "Das Aufzeichnen Ihrer React.js-Website oder -App ist mit Matomo einfach, selbst für Single-Page-Anwendungen.",
+        "SiteWithoutDataReactFollowStepCompleted": "%1$sHerzlichen Glückwunsch!%2$s Sie haben den Matomo Analytics-Tracking-Code erfolgreich über den Matomo Tag Manager installiert. Um zu überprüfen ob Besucher aufgezeichnet werden, besuchen Sie Ihre Website und prüfen Sie, ob diese Daten in Ihrer Matomo-Instanz sichtbar sind.",
+        "ReactFollowStep16": "Fügen Sie den %1$sMatomo Tag Manager JS%2$s-Code in Ihre %3$s (oder andere relevante Dateien) ein. Wir empfehlen, dies mit der Methode \"%4$s\" zu tun. Das folgende Beispiel zeigt, wie Sie den %1$sMatomo Tag Manager JS%2$s-Code in einer \"%5$s\"-App in %6$s hinzufügen.",
+
+        "SPAFollowStep1": "Wählen Sie den richtigen Container für Ihre Website aus.%1$sHinweis:%2$s Matomo konfiguriert automatisch einen Container, wenn Sie eine Website hinzufügen. Standardmäßig ist dieser Container in den unten stehenden Feldern ausgewählt. Wenn Sie einen einzelnen Container für mehrere Websites verwenden möchten, stellen Sie sicher, dass Sie diesen spezifischen Container auswählen. Sie können %3$shier Ihre Container verwalten%4$s.",
+        "SPAFollowStep2": "Navigieren Sie zu den %1$sTrigger des Containers%2$s.",
+        "SPAFollowStep3": "Überprüfen Sie, ob bereits ein Trigger vom Typ %1$s vorhanden ist. Wenn Sie den %2$s-Trigger nicht sehen, folgen Sie %3$sdieser Anleitung%4$s.",
+        "SPAFollowStep5": "Klicken Sie auf %1$s im Abschnitt %2$s.",
+        "SPAFollowStep7": "Navigieren Sie zu den %1$sTags des Containers%2$s.",
+        "SPAFollowStep8": "Klicken Sie für den %1$s-Tag mit dem Typ %2$s auf das Bearbeiten-Symbol. Wenn Sie den %3$s-Tag nicht sehen, folgen Sie %4$sdieser Anleitung%5$s.",
+        "SPAFollowStep9": "In der Sektion \"%1$s\" setzen Sie das Feld \"%2$s\" auf den folgenden Wert: %3$s{{PageTitle}}%4$s",
+        "SPAFollowStep10": "Setzen Sie das Feld \"%1$s\" wie folgt:",
+        "SPAFollowStep10a": "Wenn Ihre SPA oder PWA ein %1$s#%2$s in der URL verwendet, um zu verschiedenen Seiten zu navigieren, setzen Sie das Feld \"%3$s\" auf diesen Wert: %1$s{{PageOrigin}}/{{PageHash}}%2$s",
+        "SPAFollowStep10b": "Wenn Ihre SPA oder PWA kein # in der URL verwendet, um zu verschiedenen Seiten zu navigieren, setzen Sie das Feld \"%1$s\" auf diesen Wert: %2$s{{PageUrl}}%3$s",
+        "SPAFollowStep11": "In der Sektion \"%1$s\", unter \"%2$s\", fügen Sie den \"%3$s\"-Trigger direkt unter den vorausgewählten \"%4$s\"-Trigger hinzu.",
+        "SPAFollowStep13": "Klicken Sie im linken Menü auf \"%1$s\".",
+        "SPAFollowStep14": "Setzen Sie im Feld \"%1$s\" die Version auf beispielsweise %2$s{{PageUrl}}%3$s",
```

Comment:
Is `{{PageUrl}}` correct here? Isn't the translations used for the version?

Reply:
You got eagle eyes man, great catch. Probably an oversight when doing bulk replacements.